<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>survey</class>
 <widget class="QDialog" name="survey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="surveyLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1080</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1600</y>
     <width>880</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="drinkEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>880</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="drinkLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>879</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>1300</y>
     <width>880</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Save $1.00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1100</y>
     <width>880</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>900</y>
     <width>879</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1170</y>
     <width>879</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>1450</y>
     <width>200</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="bottleSizeLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>610</y>
     <width>879</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>780</y>
     <width>881</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="ozRB_12">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color:       transparent;
    color:                  white;
	
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:      white;
    border:                 2px solid white;
}</string>
        </property>
        <property name="text">
         <string>12oz (355mL)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ozRB_16">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color:       transparent;
    color:                  white;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:      white;
    border:                 2px solid white;
}</string>
        </property>
        <property name="text">
         <string>16oz (473mL)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="ozRB_24">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color:       transparent;
    color:                  white;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:      white;
    border:                 2px solid white;
}</string>
        </property>
        <property name="text">
         <string>24oz (710mL)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ozRB_NA">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color:       transparent;
    color:                  white;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:      white;
    border:                 2px solid white;
}</string>
        </property>
        <property name="text">
         <string>None of the Above</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="ozRB_DK">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color:       transparent;
    color:                  white;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:      white;
    border:                 2px solid white;
}</string>
        </property>
        <property name="text">
         <string>Don't Really Know</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hiddenButton">
        <property name="text">
         <string>hidden</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
